docs/books/admin_guide/06-users.md

```diff
@@ -798,6 +815,7 @@ This file is modified by the command `useradd -D` (`useradd -D` entered without
 In order for the `useradd` command to retrieve the value of the `GROUP` field from the `/etc/default/useradd` file, you must specify the `-N` option.
 
 Example:
+
```

Comment:
proposal:
**Umask;**
instead of
**Umask ;**

Reply:
Will fix before merge.